build.gradle.kts

```diff
@@ -108,6 +109,10 @@ versionIceberg = System.getProperty("nessie.versionIceberg", versionIceberg)
 // Allow overriding the Nessie version used by integration tests that depend on Iceberg
 versionClientNessie = System.getProperty("nessie.versionClientNessie", versionClientNessie)
 
+// Allow overriding the postres container version used by integration tests
+versionPostgresContainerTag =
```

Comment:
This change doesn't work and is redundant. Please remove and reverify.

Reply:
okay ajantha. i will remove it and reverify.

Reply:
I believe some problem in revert. It reverted other changes too. Please recheck.